src/game/rdr/data/ObjModels.hpp

```diff
@@ -0,0 +1,322 @@
+#pragma once
+#include <string>
+#include <unordered_set>
+
+
+namespace YimMenu::Data
+{
+	struct ObjModelInfo
+	{
+		std::string model;
+		std::string type;
+	};
+
+	const ObjModelInfo g_ObjModels[]{
```

Comment:
Consistent uppercase casing should be used here, eventually.

Reply:
Indeed, however, this object list is MASSIVE though, so converting all of that to uppercase would be a bit of a pain in the ass.  Rather than doing that, couldn't we simply read/display it in uppercase in the UI?

Reply:
> Why not make a tool to capitalize them? Might as well do the rest of the joaated lists as well 😉

I don't know how to do such a thing lmao.  here is the object list https://raw.githubusercontent.com/femga/rdr3_discoveries/master/objects/object_list.lua